scripts/deploy-frontend.sh

```diff
@@ -13,6 +13,7 @@ CF_SPACE=${5}
 ENVIRONMENT=${6}
 
 env=${CF_SPACE#"tanf-"}
+frontend_app_name=$(echo $CGHOSTNAME_FRONTEND | cut -d"-" -f3)
```

Comment:
Is this no longer used, can this be deleted?

Reply:
It will be needed when we migrate PLG to production for routing. I left it because I made PLG a public URL for the time being to expedite debugging and ease of access.